src/plugins/intel_cpu/src/ngraph_transformations/convert_to_cpu_specific_opset.hpp

```diff
@@ -19,6 +19,7 @@
 #include "transformations/utils/utils.hpp"
 #include "rnn_sequences_optimization.hpp"
 #include "transformations/common_optimizations/reshape_sequence_fusion.hpp"
+#include "convert_fq_rnn_to_quantized_rnn.hpp"
```

Comment:
Should we register this transformation in ConvertToCPUSpecificOpset manager? If no, we can remove this include

Reply:
Thanks. Removed
The onvert_fq_rnn_to_quantized_rnn transformation has to be executed before snippets since they convert FQ operations to Subgraph otherwise.